### DISCUSSION TITLE
Batch Jobs Management

### DISCUSSION THREAD

**Discussion by RegisWeduLabs** [medusajs/medusa#2782] [Feature Requests]
Currently, the only way we can see batch progress is trough the notification within the admin portal.  It would be great to have an area in the Admin portal where we can view all batch jobs and interact/cancel/delete pending/stuck/processed batch jobs.

If a batch job gets stuck (ie, importing products), we need to stop the instance and manually remove the batch jobs from the batch_job table before being able to re-import or create a new batch job.  While it does not fix the underlying issue, at the very least the end user could have some sort of control over the batch jobs.

**Comment by olivermrbl** [comment]
This is an excellent suggestion @RegisHubelia. We have been discussing creating a "notifications hub" in our admin system to give you full visibility of all notifications, including batch jobs. Though this is still only in the ideation phase, and I don't think it will be a priority in the near future. 

As a simple first iteration, we could expand upon the current set of interactions in the notifications side bar.
